x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/enrichments/utils/transforms.ts

```diff
@@ -23,7 +23,7 @@ export const applyEnrichmentsToEvents: ApplyEnrichmentsToEvents = ({
   enrichmentsList,
   logger,
 }) => {
-  const mergedEnrichments = mergeEnrichments(enrichmentsList);
+  const mergedEnrichments: EventsMapByEnrichments = mergeEnrichments(enrichmentsList);
```

Comment:
do we need `: EventsMapByEnrichments` here? it looks like `mergeEnrichments` return type is explicitly set to `EventsMapByEnrichments`, so the type should be deduced automatically?